docs/getting-started/getting-started-http-07-request-response-bodies.md

```diff
@@ -0,0 +1,58 @@
+---
+id: getting-started-http-07-request-response-bodies
+title: Request and Response Bodies
+---
+
+# Request and Response Bodies
+
+Request and response bodies can be declared explicitly using the `@body` decorator. This decorator helps to clearly indicate which part of the model is the body of the request or response. While it may not change the API's functionality, it provides several benefits:
+
+1. **Clarity and readability**: Using the `@body` decorator makes it explicit which part of the model is intended to be the body. This can improve the readability of the code, making it easier for developers to understand the structure of the API.
+2. **Consistency**: Applying the `@body` decorator consistently across your API definitions can help maintain a uniform style. This can be particularly useful in larger projects with multiple contributors.
+3. **Tooling and documentation**: Some tools and documentation generators may rely on the `@body` decorator to produce more accurate and detailed outputs. By explicitly marking the body, you ensure that these tools can correctly interpret and document your API.
```

Comment:
Not sure those points are really accurate.

Body allows you to specify an explicit property that represent exactly the body. But other uses are just as valid

Reply:
should also mention `@bodyRoot`

Reply:
Added info on @bodyRoot, although I left the other points intact. I'm open to revising if you think there are other points that are more relevant.